web/package.json

```diff
@@ -5,12 +5,14 @@
   "scripts": {
     "clean": "rm -rf ./dist",
     "build": "npm run clean && NODE_ENV=production npm run ts-node ./node_modules/.bin/webpack",
+    "build:fast": "npm run ts-node-fast ./node_modules/.bin/webpack",
```

Comment:
actually, what makes it faster is perhaps more the removal of `npm run clean && NODE_ENV=production` than the `ts-node` options ....

Reply:
yes, the `NODE_ENV=production` part makes the difference:
- `21677ms` without it
- `51624ms` with it

Using `npm run clean` doesn't change a lot the total time (around 2000ms) so I would suggest to keep it as it ensure the dist folder is recreated from scratch and avoid any cached bundle / manifest / asset

Reply:
going deeper the `production` is slower mainly because of `module.exports.optimization.minimize`